custom_model_runner/datarobot_drum/resource/components/Python/everything_predictor/everything_predictor.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from datarobot_drum.drum.common import LOGGER_NAME_PREFIX
+from datarobot_drum.drum.language_predictors.python_predictor.python_predictor import (
+    PythonPredictor,
+)
+from datarobot_drum.drum.language_predictors.java_predictor.java_predictor import JavaPredictor
+from datarobot_drum.drum.common import RunLanguage
+
+from mlpiper.components.connectable_component import ConnectableComponent
+
+
+class EverythingPredictorComponent(ConnectableComponent):
+    def __init__(self, engine):
+        super(EverythingPredictorComponent, self).__init__(engine)
+        self.logger = logging.getLogger(LOGGER_NAME_PREFIX + "." + __name__)
+        self._run_language = None
+        self._predictor = None
+
+    def configure(self, params):
+        super(EverythingPredictorComponent, self).configure(params)
+        self._run_language = RunLanguage(params.get("run_language"))
+
+        if self._run_language == RunLanguage.PYTHON:
+            self._predictor = PythonPredictor()
+        elif self._run_language == RunLanguage.JAVA:
+            self._predictor = JavaPredictor()
+        elif self._run_language == RunLanguage.R:
```

Comment:
What if the language does not match java,python or R... ?

Reply:
I added else.
But this is internal code: we determine the language and pass further, so any error will be caught during development.